src/content/tools/pub/package-layout.md

```diff
@@ -488,6 +490,22 @@ documentation generators, or other bits of automation.
 Unlike the scripts in `bin`, these are *not* for external users of the package.
 If you have any of these, place them in a directory called `tool`.
 
+## Hooks
+
+```plaintext
+enchilada/
+  hook/
+    build.dart
+```
+
+Packages can define hooks to be invoked by the Dart and Flutter SDK.
```

Comment:
I think we should link to something that has more details here. Even if we don't have docs we can link to, can't we link to an issue or similar?

Reply:
Yes please. Otherwise developers will need to find details on their own or potentially just try adding file.

Also, is the usage of `/hook` still experimental in a sense that this could change or the functionality dropped, or perhaps only available when a certain experiment is enabled or on a certain branch? Whatever state, consider making that clear to avoid any incorrect assumptions of support.

Reply:
Maybe it's better to just point to https://github.com/dart-lang/native/blob/main/pkgs/native_assets_cli/README.md, this will be updated on a more continuous basis. WDYT?

Reply:
Linking to README SGTM